.idea/exercise.iml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
One general comment about the project, kind of nitpicky but still relevant IMO. We should add a .gitignore file to ignore folders such as .idea. No big deal